pkg/util/input.go

```diff
@@ -52,3 +53,21 @@ func AskForConfirmation(message string) bool {
 		" Do you want to continue [y/N]?")
 	return strings.ToUpper(userConfirmation) == "Y"
 }
+
+// Retrive password from OS native credential provider/keychain
+func GetPasswordKeyring(username string) string {
+	password, err := keyring.Get("minishift", username)
+	if err != nil {
+		return ""
+	}
+	return password
+}
+
+// Store the registration password in OS native keychain
+func SetPasswordKeyring(username, password string) error {
+	err := keyring.Set("minishift", username, password)
```

Comment:
Instead of putting `minishift` can you create a constant which should be `ProjectName` and value would be `minishift` and cosume it? you can create it in `pkg/minikube/constants/constants.go`

Reply:
Looks like there is already a constant `BinaryName` in `pkg/minishift/constants` going to use that.